client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTableRow/InvestigationTableRow.tsx

```diff
@@ -230,7 +230,8 @@ const InvestigationTableRow = ({
                     />
                 );
             case TableHeadersNames.age:
-                return indexedRow[cellName as keyof typeof TableHeadersNames] || '-';
+                const age = indexedRow[cellName as keyof typeof TableHeadersNames]
+                return age !== null ? age : '-';
```

Comment:
can't you use indexedRow[cellName as keyof typeof TableHeadersNames] ?? '-', if it's undefined what do you want to return?

Reply:
thanks :) changed